### PR TITLE
Feature/pf 952 aqts2020.4

### PR DESCRIPTION
**Pull request by DougSchmidt-AI**

### CODE REVIEW

**Comment by DougSchmidt-AI** [comment]
@josef-mixan-ai I just ran the script to regenerate against aqts-pg

```sh
$ cd src/Aquarius.Client/TimeSeries/Client
$ ./create_service_models.sh aqts-pg
Determining AQUARIUS Server version ...
Generating  ...
unix2dos: converting file ./ServiceModels/Publish.cs to DOS format...
Complete.
Determining AQUARIUS Server version ...
Generating  ...
unix2dos: converting file ./ServiceModels/Provisioning.cs to DOS format...
Complete.
Determining AQUARIUS Server version ...
Generating  ...
unix2dos: converting file ./ServiceModels/Acquisition.cs to DOS format...
Complete.
```

That was the first commit.
The second commit was the release notes update.

File: src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs [hunk 1]
```diff
@@ -602,6 +604,21 @@ public interface IStackPositionMetadataOperation
...
+        {
+            Tags = new List<TagMetadata>{};
+        }
+
+        ///<summary>
+        ///Tags
+        ///</summary>
+        [ApiMember(DataType="array", Description="Tags")]
+        public List<TagMetadata> Tags { get; set; }
+    }
```
Comment (josef-mixan-ai) [inline src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs]: `LocationAttachment` is going to be renamed to it's previous class mane `Attachment` I believe as a result of fix in PR  https://github.com/AquaticInformatics/Aquarius/pull/15307/files

Comment (josef-mixan-ai) [reply src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs]: It's not in the nightly yet.

File: src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs [hunk 2]
```diff
@@ -7260,7 +7298,7 @@ public LocationDataServiceResponse()
             ExtendedAttributes = new List<ExtendedAttribute>{};
             LocationRemarks = new List<LocationRemark>{};
             LocationNotes = new List<LocationNote>{};
-            Attachments = new List<Attachment>{};
+            Attachments = new List<LocationAttachment>{};
```
Comment (josef-mixan-ai) [inline src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs]: This going to be reverted as per the fix mentioned.

**Review by josef-mixan-ai — COMMENTED** [review]
All looking good. Lots of changes brought by move to Open Api.
Probably worth to wait until tomorrow deploy and get the AQ-27044 fixes in in the same PR?

**Pull request merged**